db/migrations/48_remove_groups_items_propagate_access.sql

```diff
@@ -116,7 +118,7 @@ CREATE DEFINER=`algorea`@`%` TRIGGER `after_insert_items_items` AFTER INSERT ON
     INSERT INTO `groups_items_propagate`
     SELECT `id`, 'children' as `propagate_access` FROM `groups_items`
     WHERE `groups_items`.`item_id` = NEW.`parent_item_id`
-    ON DUPLICATE KEY UPDATE propagate_access = IF(propagate_access = 'done', 'children', propagate_access);
+    ON DUPLICATE KEY UPDATE propagate_access = IF(propagate_access != 'self', 'children', propagate_access);
```

Comment:
I don't get your modification. Of course, if `propagate_access != 'self'` so is equals to 'children', put 'children", so put the same value in any case ... so it is just a "on duplicate key -> ignore"

same x3

Reply:
fixed